examples/swapi/src/AddPerson.re

```diff
@@ -0,0 +1,37 @@
+let ste = ReasonReact.stringToElement;
+
+module AddPerson = [%graphql {|
+    mutation addPerson($age: Int!, $name: String!) {
+        createPerson(age: $age, name: $name) {
+          name
+        }
+      }
+|}];
+
+let component = ReasonReact.statelessComponent("AddPerson");
+
+module AddPersonMutation = ReasonApollo.CreateMutation(AddPerson);
+
+let make = (_children) => {
+    ...component,
+    render: _self => {
+        let dic = Js.Dict.empty();
+        Js.Dict.set(dic, "age", Js.Json.number(24 |> float_of_int));
+        Js.Dict.set(dic, "name", Js.Json.string("Bob"));
+        let variables = Js.Json.object_(dic);
+
+        <AddPersonMutation>
+            ...(
+                (mutation, _)  => {
+                    <button onClick=((_) => {
+                        mutation({
+                            "variables": Js.Nullable.return(variables),
+                            "refetchQueries": [|"getAllPersons"|]
+                        }) |> ignore;
```

Comment:
what do you think to add a Reason layer so the user does not call the actual JS version directly?
so we could `mutate` would have the type `(~variables: option(Js.Json.t)=?, ~refetchQueries: option(string)=?, unit) => Js.Promise.t(Query.t)` to call `mutate(~variables=variables, ())` or `mutate(~variables, ~refetchQueries=["FooQuery"], ())`

Reply:
Yeah I was thinking the same when using it in this example. I'll add it now

Reply:
Ok, that's pushed